PeerioTechnologies/peerio-desktop
# style: new chat user search error

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/6632/desktop-update-invite-send-button-on-room-dm-creation-page
 
#### Testing instructions  
Simple styling fix for new chat (both Room & DM).

* On unsuccessful user search, the "user not found" message is moved outside of the scroll container, but below the grey border of chat-creation-header
  * This allows the "Send" button to be properly aligned with the "Open" button
* Fixed styling for affirmative button disabled state
  * Previously had white text on transparent background, making it invisible on this view
  * The only other instance of this style of button is on login, which has been updated to use the old style (which is now unique to the login page, as it should be)

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
